drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -36,36 +36,36 @@
 <script id="item-input-tpl" type="text/html">
     <div class="item">
         <div class="row">
-            <label>Text</label>
+            <label>{{i18n "Text"}}</label>
             <input type="text" class="item-text" value="{{ displayName }}"/>
-            <label>Zone</label>
+            <label>{{i18n "Zone"}}</label>
             <select class="zone-select">{{ dropdown }}</select>
             <a href="#" class="remove-item hidden">
                 <div class="icon remove"></div>
             </a>
         </div>
         <div class="row">
-            <label>Background image URL (alternative to the text)</label>
+            <label>{{i18n "Background image URL (alternative to the text)"}}</label>
             <textarea class="background-image">{{ backgroundImage }}</textarea>
         </div>
         <div class="row">
-            <label>Success Feedback</label>
+            <label>{{i18n "Success Feedback"}}</label>
             <textarea class="success-feedback">{{ feedback.correct }}</textarea>
         </div>
         <div class="row">
-            <label>Error Feedback</label>
+            <label>{{i18n "Error Feedback"}}</label>
             <textarea class="error-feedback">{{ feedback.incorrect }}</textarea>
         </div>
         <div class="row">
-            <label>Width (px - 0 for auto)</label>
+            <label>{{i18n "Width (px - 0 for auto)"}}</label>
```

Comment:
The formatting of this string may be confusing for translators - I suggest changing it to `"Width in pixels (0 for auto)"`